packages/nodes-base/nodes/BambooHR/v1/actions/reports/get/description.ts

```diff
@@ -0,0 +1,64 @@
+import {
+	ReportsProperties,
+} from '../../Interfaces';
+
+export const reportsGetDescription: ReportsProperties = [
+	{
+		displayName: 'Report ID',
+		name: 'id',
+		type: 'string',
+		required: true,
+		displayOptions: {
+			show: {
+				operation: [
+					'get',
+				],
+				resource: [
+					'reports',
+				],
+			},
+		},
+		default: '',
+		description: 'ID of the Report. You can get the report number by hovering over the report name on the reports page and grabbing the ID.',
```

Comment:
```suggestion
		description: 'ID of the report. You can get the report number by hovering over the report name on the reports page and grabbing the ID.',
```
In case this ID is also available in the URL of the report, we also allow users to grab and paste the URL into the field and we extract it from there, for better UX.

Reply:
Updated.